jbmc/unit/java_bytecode/java_bytecode_parse_generics/parse_bounded_generic_inner_classes.cpp

```diff
@@ -68,6 +68,28 @@ SCENARIO(
             to_struct_type(class_symbol.type), "elem");
         require_type::require_java_generic_parameter(
           elem.type(), boundedinner_name + "::NUM");
+
+        std::string method_name =
+          boundedinner_name + ".f:(Ljava/lang/Number;)V";
+        REQUIRE(new_symbol_table.has_symbol(method_name));
+        THEN("The method parameter type should respect its bound")
+        {
+          const symbolt &method_symbol =
+            new_symbol_table.lookup_ref(method_name);
+          const code_typet &method_type =
+            require_type::require_code(method_symbol.type);
+          const code_typet::parametert &param =
+            require_type::require_parameter(method_type, "x");
+          const typet &param_type = param.type();
+          REQUIRE(param_type.id() == ID_pointer);
+          REQUIRE(param_type.subtype().id() == ID_symbol);
+#if 0
```

Comment:
Rather than `if 0`, I would just add a comment with a TODO referring to the relevant ticket number that should fix the bounds - TG-1286 perhaps. Similarly in the other tests.

Reply:
Okay. I was trying to save someone time in future working out what tests to write when TG-1286 is fixed.